utils/rlwrap/Makefile

```diff
@@ -0,0 +1,51 @@
+# Copyright (C) 2019-2025 Jerônimo Pellegrini <j_p@aleph0.info>
+#
+# This file is free software, licensed under the GNU General Public License v3
+# or later.
+# See /LICENSE for details
```

Comment:
Consider adding SPDX license header instead, and I don't think GPL 2 and 3 are compatible.

Reply:
Fixed